index.markdown

```diff
@@ -118,7 +118,7 @@ features:
     - title: <b>Cloud-based computing</b>
       description: <a href="https://discourse.slicer.org/t/how-to-run-slicer-on-the-cloud-and-access-in-a-web-browser/16401">3D Slicer in web browser</a>, <a href="https://github.com/Slicer/SlicerDocker">Docker container</a>, or <a href="https://discourse.slicer.org/t/run-slicer-in-your-web-browser-as-a-jupyter-notebook-or-as-a-full-application/11569">Jupyter notebook kernel</a>. <b><a href="https://mybinder.org/v2/gh/Slicer/SlicerNotebooks/master?filepath=SlicerWeb.ipynb">Click here to run Slicer in your web browser now</a></b> (using free Binder service, may take 1-2 minutes to start).
     - title: <b>Extensions</b>
-      description: Over 150 extensions that can be installed and used with the application using the <a href="https://slicer.readthedocs.io/en/latest/user_guide/extensions_manager.html">3D Slicer App Store</a>.
+      description: Over 150 extensions that can be installed and used with the application using the <a href="https://extensions.slicer.org">3D Slicer App Store</a>.
```

Comment:
```suggestion
      description: Over 150 extensions that can be installed and used with the application using the <a href="https://slicer.readthedocs.io/en/latest/user_guide/extensions.html">3D Slicer App Store</a>.
```

After adding the top-level `Extensions` section, we could revisit:
* https://github.com/Slicer/Slicer/pull/8453

Reply:
I noticed the Slicer readthedocs page points to an old stable version of browsing extensions (rev 30822)

https://github.com/Slicer/Slicer/blob/8bb7142be207d5f838e3708dd0c7b6d51d3e58c7/Docs/user_guide/extensions_manager.md?plain=1#L7